src/main/resources/assets/emt/lang/en_US.lang

```diff
@@ -251,6 +252,10 @@ emt.rsname.Shield Focus=Shield Focus
 emt.tag.Shield Focus=S.H.I.E.L.D.
 emt.text.Shield Focus=With some experimentation with the Portable Hole, you have found out more about the transportation of objects to alternate dimensions. You believe you can reverse these effects to summon a swath of blocks from a dimension full of matter to shield you from any oncoming disaster. Caution is to be taken when using this focus, as when bringing these blocks into our dimension, it will transpose any blocks in the way into the dimension it came from, never to return again. You will also be unable to move while this shield is in existence.
 
+emt.rsname.Wand Focus: Maintenance=Maintenance Focus
+emt.tag.Want Focus: Maintenance=Who needs a toolbox when you got Magic
```

Comment:
* Who needs a toolbox when you have magic?

Alternatively: Who needs tools when magic exists?